test/testcafe/framework/page-objects/EnrollOktaVerifyPageObject.js

```diff
@@ -1,66 +1,160 @@
+import { within } from '@testing-library/testcafe';
 
-const FORM_INFOBOX_ERROR = '[data-se="o-form-error-container"] [data-se="callout"]';
 const FORM_INFOBOX_ERROR_TITLE = '[data-se="o-form-error-container"] [data-se="callout"] > h3';
+const FORM_INFOBOX_ERROR_TITLE_V3 = '[data-se="callout"] > div > h2';
+const CANT_SCAN_BUTTON_SELECTOR = 'button[aria-label="Setup without scanning a QR code."]';
+const FORM_SELECTOR = '[data-se="o-form-explain"]';
 
 export default class EnrollOktaVerifyPageObject extends BasePageObject {
   constructor(t) {
     super(t);
   }
 
-  hasEnrollViaQRInstruction() {
+  async hasEnrollViaQRInstruction() {
+    if (userVariables.v3) {
+      if (await this.form.elementExist('.qrImg')) {
+        const qrInstruction = await this.getNthInstructionBulletPoint(0);
+        return qrInstruction.includes('Okta Verify');
+      } else {
+        return false;
+      }
+    }
     return this.form.elementExist('.qrcode-info');
   }
 
   hasQRcode() {
+    if (userVariables.v3) {
+      return this.form.elementExist('.qrImg');
+    }
     return this.form.elementExist('.qrcode');
   }
 
-  hasEnrollViaEmailInstruction() {
+  async hasEnrollViaEmailInstruction() {
+    if (userVariables.v3) {
+      if (await this.form.elementExist(FORM_SELECTOR)) {
+        const instruction = await this.form.getElement(FORM_SELECTOR).innerText;
+        return instruction.includes('email');
+      } else {
+        return false;
+      }
+    }
     return this.form.elementExist('.email-info');
   }
 
-  hasEnrollViaSmsInstruction() {
+  async hasEnrollViaSmsInstruction() {
+    if (userVariables.v3) {
+      if (await this.form.elementExist(FORM_SELECTOR)) {
+        const instruction = await this.form.getElement(FORM_SELECTOR).innerText;
+        return instruction.includes('SMS');
+      } else {
+        return false;
+      }
+    }
     return this.form.elementExist('.sms-info');
   }
 
-  getQRInstruction() {
+  getQRInstruction(index) {
+    if (userVariables.v3) {
+      return this.getNthInstructionBulletPoint(index);
+    }
     return this.getTextContent('.qrcode-info');
   }
 
+  async getNthInstructionBulletPoint(index) {
+    const listItems = within(this.form.el).getAllByRole('listitem');
+    return listItems.nth(index).innerText;
+  }
+
   getEmailInstruction() {
+    if (userVariables.v3) {
+      return this.getTextContent('[data-se="o-form-explain"]'); 
+    }
     return this.getTextContent('.email-info');
   }
 
   getSmsInstruction() {
+    if (userVariables.v3) {
+      return this.getTextContent('[data-se="o-form-explain"]'); 
+    }
     return this.getTextContent('.sms-info');
   }
 
   getSwitchChannelText() {
+    if (userVariables.v3) {
+      return this.getTextContent(CANT_SCAN_BUTTON_SELECTOR);
+    }
     return this.getTextContent('.switch-channel-link');
   }
 
   async clickSwitchChannel() {
+    if (userVariables.v3) {
+      await this.form.clickElement(CANT_SCAN_BUTTON_SELECTOR);
```

Comment:
Wouldn't work in v2 since it's a link and not a button. 
<img width="1013" alt="Screenshot 2023-04-19 at 2 29 09 PM" src="https://user-images.githubusercontent.com/27521424/233204380-265336bd-9ad3-4428-980e-5f30a3ee0b37.png">

Reply:
And for some reason, `await this.form.clickButton('Can\'t scan?');` doesn't work in v3 either.

Reply:
yeah the `clickButton` method is looking for a button with a name of "Cant scan?" , but in this case the "name" of that button is its `aria-label`, which is "Setup without scanning a QR code.".  So if you use 
`await this.form.clickButton('Setup without scanning a QR code.');` that should work.